mixpanel/__init__.py

```diff
@@ -251,6 +251,22 @@ def people_unset(self, distinct_id, properties, meta=None):
             '$unset': properties,
         }, meta=meta)
 
+    def people_remove(self, distinct_id, properties, meta=None):
+        """Remove a value from the list associated with a property.
```

Comment:
Just to remove any possible confusion:

```suggestion
        """Permanently remove a value from the list associated with a property.
```